crates/solidity/outputs/cargo/tests/src/bindings_output/generated/mod.rs

```diff
@@ -0,0 +1,43 @@
+// This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+use semver::Version;
+mod enums;
+mod expressions;
+mod lexical;
+mod structs;
+
+pub const VERSION_BREAKS: [Version; 33] = [
```

Comment:
I wonder if we can generate this once, instead of 3 times now in this crate.

Reply:
Yes, I like it. I'll change it.